Automattic/jetpack
# Jetpack Components: Add Status Component

Adds a small component for displaying status indicators.

### Other information:


## Jetpack product discussion



p1HpG7-lmA-p2

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Check out this branch
* Run `jetpack install -r`
* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Check out the new "Status" stories found under "Components".

## Screenshots

<img width="826" alt="Screenshot 2023-04-19 at 4 33 05 PM" src="https://user-images.githubusercontent.com/10933065/233214357-f88c599a-715c-4591-8f24-072b31ddc021.png">
